v-next/hardhat-ignition-ethers/src/types.ts

```diff
@@ -0,0 +1,56 @@
+import type {
+  IgnitionModuleResult,
+  StrategyConfig,
+  IgnitionModule,
+  DeploymentParameters,
+  DeployConfig,
+  NamedArtifactContractDeploymentFuture,
+  NamedArtifactContractAtFuture,
+} from "@ignored/hardhat-vnext-ignition-core";
+import type { Contract } from "ethers/contract";
+
+export interface EthersIgnitionHelper {
```

Comment:
We follow the same pattern of splitting out an interface and exposing that as the public type, the impl moving to internal.